### ISSUE TITLE
MInor character encoding issue

### ISSUE THREAD

**Issue by dosumis** [hubmapconsortium/ccf-validation-tools#178]
Wondering if this comes from our scripts or from source data not being UTF-8:

https://github.com/hubmapconsortium/ccf-validation-tools/blob/master/logs/new_cl_terms_Eye.tsv

has  "Gr√ºnert"

**Comment by anitacaron** [comment]
This comes from the ASCT+B table.

![Screenshot 2022-10-04 at 11 37 08](https://user-images.githubusercontent.com/2208124/193799507-a09e068e-84cb-4a13-ad04-427b939a6294.png)

**Comment by bherr2** [comment]
Yeah, it's definitely a unicode thing. I wish there was an open format that handled this nicely (csv doesn't dictate the encoding and it's not always obvious what the encoding is in depending on platform) but was not too difficult to parse.

**Comment by dosumis** [comment]
I think it should be possible to write tests to catch these --> logs, and then have someone on the team correct by hand.
